databricks/koalas/indexes.py

```diff
@@ -108,13 +108,26 @@ class Index(IndexOpsMixin):
     Index(['a', 'b', 'c'], dtype='object')
     """
 
-    def __init__(self, data: Union[DataFrame, list], dtype=None, name=None) -> None:
+    def __new__(cls, data: Union[DataFrame, list], dtype=None, name=None, names=None):
+        if isinstance(data, list) and all([isinstance(item, tuple) for item in data]):
+            # MultiIndex
+            return MultiIndex.from_tuples(data, names=names)
+        else:
+            # Index
+            return super().__new__(cls)
+
+    def __init__(self, data: Union[DataFrame, list], dtype=None, name=None, names=None) -> None:
```

Comment:
How does it work if both `name` and `names` are set? Looks like we should throw an exception properly.

Reply:
Thanks, @HyukjinKwon . I'll manage this.